spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -77,4 +77,9 @@ private SparkReadOptions() {}
 
   // Timestamp in milliseconds; start a stream from the snapshot that occurs after this timestamp
   public static final String STREAM_FROM_TIMESTAMP = "stream-from-timestamp";
+
+  // maximum file per micro_batch
+  public static final String MAX_FILES_PER_MICRO_BATCH = "max-files-per-micro-batch";
```

Comment:
given this is a streaming feature, can we use `streaming-` prefix, and name it something like `streaming-max-files-per-batch`?